frontend/src/components/VariantListPage/VariantListPage.tsx

```diff
@@ -179,8 +193,20 @@ const useVariantListAnnotation = (variantList: VariantList) => {
           variant_notes: Record<VariantId, string>;
           variant_calculations: VariantListCalculations;
           include_homozygotes_in_calculations: boolean;
+          taggedGroups: TaggedGroups;
         }) => {
           const selectedVariants = new Set(annotation.selected_variants);
+          const taggedGroups: {
+            A: { displayName: string; variantList: Set<VariantId> };
+            B: { displayName: string; variantList: Set<VariantId> };
+            C: { displayName: string; variantList: Set<VariantId> };
+            D: { displayName: string; variantList: Set<VariantId> };
```

Comment:
An explicit type def here is a bit surpising, is there another existing type def we can use? 

If not, do we even need this? TypeScript is pretty good at inferring type from creation of an object, and we pretty explicitly define this object below.